src/util/functionsMap.ts

```diff
@@ -0,0 +1,40 @@
+import { Signer } from "@ethersproject/abstract-signer"
+import { TransactionReceipt } from "@ethersproject/abstract-provider"
+import { Layer } from "../util/provider"
+import { Metadata } from "../util/metadata"
+import { closePosition, openPosition, OpenPositionArg, ClosePositionArg } from "../exec/contract/ClearingHouse"
+
+export interface Actions {
+    action: string
+    args: EmptyArg
+}
+
+export interface EmptyArg {
```

Comment:
`BaseArgs` would be better? it kind of the base args for all argument which need providing `verify()` function